app/routes/meetings/publish/publication-actions/detail.js

```diff
@@ -12,8 +12,16 @@ export default class MeetingsPublishPublicationActionsDetailRoute extends Route
       },
     );
     const signedResource = await log.signedResource;
-    if (signedResource) {
+    if (signedResource?.content) {
       return signedResource.content;
+    } else if (signedResource?.file) {
+      const fileMeta = await signedResource.file;
+      const fileReq = await fetch(fileMeta.downloadLink);
+      if (fileReq.ok) {
+        return fileReq.text();
+      } else {
+        return `<div class="au-c-alert au-c-alert--warning"><p>Error fetching file contents: ${fileReq.statusText}</p></div>`;
+      }
     } else {
       const publishedResource = await log.publishedResource;
       return publishedResource.content;
```

Comment:
We probably also need to check for the `file` relationship here.

Reply:
Good spot, yes.